docs/design_in_detail.md

```diff
@@ -170,4 +170,4 @@ The built-in TTL mechanism of KubeBrain exists as a component of the "Compact" m
 
 **Why not implement the event mechanism based on binlog？**
 
-First of all, binlog is not a feature that all distributed kv databases must have. For distributed kv databases, enabling binlog will greatly increase write latency and reduces system throughput. In the K8s system, the event generated when the metadata is changed is used to incrementally update the cache. When the KubeBrain node crashes and the memory event is lost, the cache can be resynchronized through the list watch through the API Server, which can tolerate some event loss. Of course, subsequent projects will continue to optimize in this regard.
\ No newline at end of file
+First of all, binlog is not a feature that all distributed kv databases must have. For distributed kv databases, enabling binlog will greatly increase write latency and reduces system throughput. In the K8s system, the event generated when the metadata is changed is used to incrementally update the cache. When the KubeBrain node crashes and the memory event is lost, the cache can be resynchronized through the list watch through the API Server, which can tolerate some event loss. Of course, subsequent projects will continue to optimize in this regard.
```

Comment:
Sorry, i can't find the changes in this line. Can you explain the changes?

Reply:
It appends an extra `\n` at the end of the line. :P

![image](https://user-images.githubusercontent.com/20221408/181789004-cf730d63-68b1-4bd1-9589-d91c3d795c1c.png)

ref: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
Got It.